src/components/NavMenu/NavMenu.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Link } from "react-router-dom";
```

Comment:
The `react-router-dom` dependency isn't included in your package.json file, so it's throwing an error. You shouldn't need it though since `Link` is used in gatsby. Maybe just remove this line.